src/FormBuilder/resources/js/form-builder/src/components/canvas/FormSettingsContainer/components/Content.tsx

```diff
@@ -0,0 +1,19 @@
+import {useFormSettingsContext} from '@givewp/form-builder/components/canvas/FormSettingsContainer';
+
+function renderRoutes(routes) {
+    const [state] = useFormSettingsContext();
+
+    return routes.map((route) => {
+        if (route.childRoutes && route.childRoutes.length > 0) {
+            return renderRoutes(route.childRoutes);
+        }
+        return state.activeRoute === route.path && route.element;
+    });
+}
+export default function Content({routes}) {
```

Comment:
please add `@unreleased` tag and description

Reply:
Resolved: 6e01c462287731e3f80d3f19711d3b03e30da8ce